client/my-sites/email/email-management/email-home.jsx

```diff
@@ -82,14 +80,10 @@ class EmailManagementHome extends React.Component {
 
 			if ( ! domainHasEmail( selectedDomain ) ) {
 				return this.renderContentWithHeader(
-					<EmailProvidersComparison
-						domain={ selectedDomain }
-						isGSuiteSupported={
-							userCanPurchaseGSuite && hasGSuiteSupportedDomain( [ selectedDomain ] )
-						}
-					/>
+					<EmailProvidersComparison selectedDomainName={ selectedDomainName } />
```

Comment:
We no longer need to pass `isGSuiteSupported` to `EmailProvidersComparison` since this component  is now responsible for loading any required data.